nextflow.config

```diff
@@ -21,12 +21,12 @@ params {
     save_reference = null // Built references not saved
 
     // Main options
-    help = false
+    help = null
```

Comment:
Why is it better to use `null` than `false`?

Reply:
No idea, and probably not good anyway, since all tests are failing anyway

Reply:
ok, so looked into everything, ran everything once again locally, and found no issues.
So just relaunched the tests, and it's working...